tests/integration/models/blueprint-test.js

```diff
@@ -181,12 +180,14 @@ describe('Blueprint', function () {
     });
 
     it('finds blueprints in the ember-cli package', function () {
-      let expectedPath = path.resolve(defaultBlueprints, 'app');
+      let expectedPath = path.dirname(require.resolve('@ember/classic-app-blueprint'));
       let expectedClass = Blueprint;
 
       let blueprint = Blueprint.lookup('app');
 
-      expect(blueprint.name).to.equal('app');
+      // TODO it's really strange that a blueprint's name is defined by its folder
```

Comment:
I have vague memories that it can be overridden -- I think there are examples in ember-source?